### PR TITLE
Enhanced ConvertGraphHandler to remove data from converted graphs

### PR DESCRIPTION
**Pull request by spoenemann**
* Converted ConvertGraphHandler to Xtend
 * Added methods to remove invalid layout options and edge layouts. These methods are currently not used (see l. 148) because it might lead to bad surprises in some cases. They can be activated by uncommenting two method calls. We could consider using a separate menu entry for this variant, but I leave that as future work.
 * Added a quick fix for options that have a wrong target element.

### CODE REVIEW

File: plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    ]
+    
+    private def isValid(IProperty<?> property, Object value, EMapPropertyHolder container) {
+        // Check whether the property is a registered layout option
+        val option =
+            if (property instanceof LayoutOptionData)
+                property
+            else
+                LayoutMetaDataService.instance.getOptionData(property.id)
+        if (option === null)
```
Comment (le-cds) [inline plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend]: Personally, I'm not a big fan of leaving out the curly braces...

**Comment by le-cds** [comment]
Apart from me not being a fan of missing curly braces, I think this can be merged. :)

File: plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend (hunk 1 above)
Comment (spoenemann) [reply plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend]: I can understand that. Recently I learned that there are even [programming languages that enforce braces](https://golang.org/ref/spec#If_statements). But omitting the braces in Xtend code matches quite well the concise style of that language.

Comment (le-cds) [reply plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend]: That's interesting. I should look at Go after all... :D

Comment (spoenemann) [reply plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend]: I don't like Go very much. It's certainly better than writing C, but IMHO the language design decisions are questionable.

I found this interesting:
https://medium.com/@george3d6/the-success-of-go-heralds-that-of-rust-73cb2e4c0500

Comment (le-cds) [reply plugins/org.eclipse.elk.graph.text.ui/src/org/eclipse/elk/graph/text/ui/ConvertGraphHandler.xtend]: I'll take a look, thanks. :)

**Pull request merged**